Update README.md (#229)
robbinjanssen/home-assistant-omnik-inverter: README.md

## Manual installation

> Create a directory called `omnik_inverter` in the `<config directory>/custom_components/` directory on your Home Assistant instance. In many installations, the <config directory> is where the configuration.yaml file is located. Create the directory custom_components first if it does not exist yet.

Reviewer: You definitely don't want to have loose `<>` (HTML constructs) in a markdown file. Escape it with backticks, and do the same to the file and folder name.

Reviewer: ```suggestion
Create a directory called `omnik_inverter` in the `<config directory>/custom_components/` directory on your Home Assistant instance. In many installations, the `<config directory>` is where the `configuration.yaml` file is located. Create the directory `custom_components/` first if it does not exist yet.
```

Reviewer: Though in hindsight this paragraph doesn't even tell users how to most conveniently download the necessary folder. Assuming most users install over the cmdline, we could even suggest a copy-pastable example like:

```bash
curl -L https://github.com/robbinjanssen/home-assistant-omnik-inverter/archive/refs/heads/main.zip | bsdtar xv --strip-components=1 home-assistant-omnik-inverter-main/custom_components/omnik_inverter/
```

This should create the `custom_components/omnik_inverter/` directory structure and only unzip the files within that folder in this repo to it.

> Install this component by copying all files in `/custom_components/omnik_inverter/` folder from this repo into the new `<config directory>/custom_components/omnik_inverter/` directory you just created. 

Reviewer: ```suggestion
Install this component by copying all files in `/custom_components/omnik_inverter/` folder from this repo into the new `<config directory>/custom_components/omnik_inverter/` directory you just created.
```

│   ├── sensor.py
│   └── strings.json
```

> Make sure to set the permissions and ownership so that hass has proper access. Restart the hass service so the integration can be installed.

Reviewer: ```suggestion
Make sure to set the permissions and ownership so that Home Assistant has proper access. Restart the Home Assistant service so the integration can be installed.
```

However, this does _not_ even explain:

1. How to set permissions;
2. What permissions to set ("so that Home Assistant has proper access": what user and `rwx` does it need?).

So I don't think it adds any value.

Reviewer 2: I'm of the opinion that this doesn't even belong in the readme. By default the permissions should already be correct, if that is not the case then you have been messing around with the config or you are using an advanced install where we can assume that someone also knows how to solve it.

I also don't know of any custom integration where this was a thing 🤷🏻‍♂️
